### PR TITLE
Capstone Module 2

### PR DESCRIPTION
**Pull request by Iqbal-Elham**
## JavaScript Capstone Project
In this project we build a API-based web-app that it has homepage, comment popup, and reservation popup, so we get the movies from the API and then we can like, give comment and reserve the movies and count each of them.
### The requirements needed:
- Using the Html/CSS
- Using JavaScript ES6 and make the code asynchronous 
- Using the linter for free of errors and beautiful coding.
- Using GitHub flow.
- Documented in a professional way and record a video.
- Using webpack, and modules
- Use AAA pattern for unit tests.
- Use API documentation.

### CODE REVIEW

File: src/modules/populateData.js [hunk 1]
```diff
@@ -0,0 +1,238 @@
...
+      reserveSubmit.addEventListener('click', async (event) => {
+        const alarmFormReservations = document.querySelector('.alarm-form-reservations');
+        const nameReservation = document.getElementById('nameReservation').value;
+        const startReservation = document.getElementById('startDate').value;
+        const endReservation = document.getElementById('endDate').value;
+        if (nameReservation && startReservation && endReservation) {
+          alarmFormReservations.innerHTML = '';
+          const regEx = /^\d{4}-\d{2}-\d{2}$/;
+          if (startReservation.match(regEx) && endReservation.match(regEx)) {
+            alarmFormReservations.innerHTML = 'Reservation Completed';
```
Comment (Owusu-Desmond) [inline src/modules/populateData.js]: - [ OPTIONAL] Most times successful messages have the color to be green and errors have their colors to be red so I think it is better if you make the `Reservation Completed` alert her have green color to differentiate the itself from the error messages:blush: 
Screenshot from 2023-01-05 23-14-01

File: src/style.css [hunk 2]
```diff
@@ -0,0 +1,720 @@
...
+
+.popup-section::-webkit-scrollbar {
+  width: 10px;
+}
+
+.popup-section::-webkit-scrollbar-thumb {
+  background: #f0f0f0;
+  border-radius: 2rem;
+}
+
```
Comment (Owusu-Desmond) [inline src/style.css]: - [ ] From the project requirements, you expected to adhere to this given design in terms of the data displayed on the page by comparing your design and the given design your popup does not look like the given popup page design.

 comparing the differences... 
  - the given design has no nav for the popup page but I can still see your nav clearly for your popup page
  - I can clearly see your home page cards in the popup page

| Your Design |Given Design |
| :---:   | :---: | 
| Screenshot from 2023-01-05 23-32-57 | image|

I think is better if you enlarge the popup window and blur the background to differentiate the two pages ie. (the home page and the popup page)

File: README.md [hunk 3]
```diff
@@ -0,0 +1,294 @@
...
+
+OPTIONAL SECTIONS:
+- FAQ
+
+After you're finished please remove all the comments and instructions!
+-->
+
+<div align="center">
+  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
+  <img src="./src/assets/favicon-images/murple_logo.png" alt="logo" width="150"  height="auto" />
```
Comment (Owusu-Desmond) [inline README.md:30]: - [ ] Please make sure you remove this microverse logo from your readme. It is meant to be replaced with your project logo if you have one. :pray:

**Review by Owusu-Desmond — CHANGES_REQUESTED** [review]
Hi @Iqbal-Elham, @belaymit and @dgcuenca :wave:  

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!



## Highlights
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Great working on the setup of the project ✅
📌 Good commit messages ✅
📌 Professional README file ✅

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by emmiiorji — APPROVED** [review]
Hi @Iqbal-Elham, @belaymit and @dgcuenca,

# Status: Approved ✅

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Congrats

## Good Highlights✔️

- Good commit messages
- Adhered to Gitflow
- Linter checks are passing
- Requested change has been implemented


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**